etc/release/scancode-create-pypi-wheel.sh

```diff
@@ -19,6 +19,7 @@ set -e
 
 ./configure --dev
 venv/bin/scancode-reindex-licenses
+venv/bin/scancode-cache-package-patterns
```

Comment:
What about naming this `venv/bin/scancode-reindex-package-patterns` to be consistent?